tests/integration_tests/test_upgrade.py

```diff
@@ -1,195 +1,195 @@
-    ).balanceOf(ADDRS["validator"])
-    p = json.loads(
-        cli.raw(
-            "query",
-            "ibc",
-            "client",
-            "params",
-            home=cli.data_dir,
-        )
-    )
-    assert p == {"allowed_clients": ["06-solomachine", "07-tendermint", "09-localhost"]}
+# import configparser
+# import json
+# import re
+# import subprocess
+# from pathlib import Path
+
+# import pytest
+# from dateutil.parser import isoparse
+# from pystarport import ports
+# from pystarport.cluster import SUPERVISOR_CONFIG_FILE
+
+# from .network import Ethermint, setup_custom_ethermint
+# from .utils import (
+#     ADDRS,
+#     CONTRACTS,
+#     deploy_contract,
+#     parse_events,
+#     send_transaction,
+#     wait_for_block,
+#     wait_for_block_time,
+#     wait_for_port,
+#     find_log_event_attrs,
+# )
+
+
+# def init_cosmovisor(home):
+#     """
+#     build and setup cosmovisor directory structure in each node's home directory
+#     """
+#     cosmovisor = home / "cosmovisor"
+#     cosmovisor.mkdir()
+#     (cosmovisor / "upgrades").symlink_to("../../../upgrades")
+#     (cosmovisor / "genesis").symlink_to("./upgrades/genesis")
+
+
+# def post_init(path, base_port, config):
+#     """
+#     prepare cosmovisor for each node
+#     """
+#     chain_id = "ethermint_9000-1"
+#     cfg = json.loads((path / chain_id / "config.json").read_text())
+#     for i, _ in enumerate(cfg["validators"]):
+#         home = path / chain_id / f"node{i}"
+#         init_cosmovisor(home)
+
+#     # patch supervisord ini config
+#     ini_path = path / chain_id / SUPERVISOR_CONFIG_FILE
+#     ini = configparser.RawConfigParser()
+#     ini.read(ini_path)
+#     reg = re.compile(rf"^program:{chain_id}-node(\d+)")
+#     for section in ini.sections():
+#         m = reg.match(section)
+#         if m:
+#             i = m.group(1)
+#             ini[section].update(
+#                 {
+#                     "command": f"cosmovisor start --home %(here)s/node{i}",
+#                     "environment": (
+#                         f"DAEMON_NAME=ethermintd,DAEMON_HOME=%(here)s/node{i}"
+#                     ),
+#                 }
+#             )
+#     with ini_path.open("w") as fp:
+#         ini.write(fp)
+
+
+# @pytest.fixture(scope="module")
+# def custom_ethermint(tmp_path_factory):
+#     path = tmp_path_factory.mktemp("upgrade")
+#     cmd = [
+#         "nix-build",
+#         Path(__file__).parent / "configs/upgrade-test-package.nix",
+#         "-o",
+#         path / "upgrades",
+#     ]
+#     print(*cmd)
+#     subprocess.run(cmd, check=True)
+#     # init with genesis binary
+#     yield from setup_custom_ethermint(
+#         path,
+#         26100,
+#         Path(__file__).parent / "configs/cosmovisor.jsonnet",
+#         post_init=post_init,
+#         chain_binary=str(path / "upgrades/genesis/bin/ethermintd"),
+#     )
+
+
+# def test_cosmovisor_upgrade(custom_ethermint: Ethermint):
```

Comment:
can you just `@pytest.mark.skip`? should minimize the diff and future merge conflicts

Reply:
Ah I was looking at the specific commit. If it was commented out before I think this is fine. We have upgrade test coverage on the node side.